### ISSUE TITLE
--v8-flags not passing flags to v8

### ISSUE THREAD

**Issue by kevinkassimo** [denoland/deno#2112]
e.g. previously by specifying `--expose-gc`, a `gc` function would be exposed. However, under the new flag parser, seems none of the v8 flags are working...
```
$ deno --v8-flags=--expose-gc
> gc()
<unknown>:1:0
gc()
^
Uncaught ReferenceError: gc is not defined
    at <unknown>:1:1
    at evaluate (js/repl.ts:87:34)
    at replLoop (js/repl.ts:145:13)
>
```

cc @bartlomieju

**Comment by bartlomieju** [comment]
I've spotted the problem, fixing it now

**Comment by bartlomieju** [comment]
This issue will be resolved by #2157

**Comment by ry** [comment]
closed in https://github.com/denoland/deno/pull/2157
